contracts/modules/token/TokenController.sol

```diff
@@ -24,6 +25,9 @@ contract TokenController is ITokenController, LockHandler, LegacyMasterAware {
   IAssessment public assessment;
   IGovernance public governance;
 
+  ICover public cover;
+  uint pooledStakingNXMBalance;
```

Comment:
if we track this as such, we would need to add all NXM balance changes, including deposits an wtihdrawals from staking pools here.